patches/master_patch.patch

```diff
@@ -1920,6 +1908,7 @@ index 9162bc3ba51153a494aa218007511054cdebbad7..e685ea161db22661d3a643cd37df8924
 -        element_->GetDocument()
```

Comment:
can you change this to use MUON_CHROMIUM_BUILD? It's hard to read the diff like this and so I'm a little confused about part of it

Reply:
good call, addressed in 8a54a8f2c17d4babe3d90eee35c8cd2b3be9611c